cmd/root.go

```diff
@@ -156,3 +158,57 @@ func initConfig() {
 func boolCompletions(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
 	return []string{"true", "false"}, cobra.ShellCompDirectiveNoFileComp
 }
+
+func addLogsFlags(cmd *cobra.Command) {
+	cmd.Flags().BoolP("follow", "f", false, "Continuously print new log entries")
+	cmd.Flags().Int32P("lines", "n", 0, "Number of log entries to show")
```

Comment:
```suggestion
	cmd.Flags().Uint32P("lines", "n", 0, "Number of log entries to show")
```
Should be uint right? Or do negative numbers make sense here? I forget if systemd-journal handles that

Reply:
That's actually copy-pasted from the old host logs, but right, I don't see a reason why it should be signed.